src/configs/axios.ts

```diff
@@ -3,11 +3,10 @@ import axios from 'axios';
 import { PAGE_URL } from './path';
 
 export const API = axios.create({
-  baseURL: 'https://causw-dev.du.r.appspot.com',
-
-  // process.env.REACT_APP_ENV === 'production'
-  //   ? 'https://causw-prod.du.r.appspot.com'
-  //   : 'https://causw-dev.du.r.appspot.com',
+  baseURL:
+    process.env.NODE_ENV !== 'development'
```

Comment:
이중 부정보단 production을 조건식으로 처리하는 게 좋을 것 같아요
```suggestion
    process.env.NODE_ENV === 'production'
```

Reply:
주석도 지워도 될 것 같아요